content/departments/product-engineering/process/index.md

```diff
@@ -1,7 +1,8 @@
 # Product and Engineering Processes
 
-- [Planning](planning.md)
+- [Planning philosophy](planning-philosophy.md)
+- [Planning communication](planning-communication.md)
+- [Planning process](planning-process.md)
```

Comment:
Planning process and communication might benefit from being on one page given how closely they connect to each other.

Reply:
Agree, will change this